src/gui/mzroll/pollyelmaveninterface.cpp

```diff
@@ -210,29 +215,28 @@ void PollyElmavenInterfaceDialog::call_initial_EPI_form() {
     QCoreApplication::processEvents();
 }
 
-void PollyElmavenInterfaceDialog::handleAuthentication(QString status) {
+void PollyElmavenInterfaceDialog::handleAuthentication(QString status)
+{
     if (status == "ok") {
         _loadingDialog->statusLabel->setText("Fetching your projects..");
         QCoreApplication::processEvents();
-    }
-    else {
-            _loadingDialog->statusLabel->setText("Authentication failed. Please login again.");
-            QCoreApplication::processEvents();
-            logout();
+    } else {
+        _loadingDialog->statusLabel->setText("Authentication failed. Please login again.");
+        QCoreApplication::processEvents();
+        logout();
     }
 }
 
-void PollyElmavenInterfaceDialog::handleResults(QVariantMap projectnamesIdMap){
+void PollyElmavenInterfaceDialog::handleResults(QVariantMap projectnamesIdMap)
+{
     projectNamesId = projectnamesIdMap;
     startupDataLoad();
```

Comment:
@shubhra-agrawal 
This function must have at least one argument projectNamesId, otherwise how are we going to pass information generated from the thread to main UI ?

It should be startupDataLoad(projectNamesId).